feature/bgp/admin_distance/README.md

```diff
@@ -0,0 +1,94 @@
+# RT-1.34: BGP route-distance configuration
+
+## Summary
+
+BGP default-route-distance, external-route-distance and internal-route-distance (administrative distance) configuration.
+
+## Testbed type
+
+*   https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_4.testbed
+
+## Procedure
+
+### Applying configuration
+
+For each section of configuration below, prepare a gnmi.SetBatch  with all the configuration items appended to one SetBatch. Then apply the configuration to the DUT in one gnmi.Set using the `replace` option
+
+#### Initial Setup:
+
+*   Connect DUT port-1, 2 and 3 to ATE port-1, 2 and 3
+*   Configure IPv4/IPv6 addresses on the ports
+*   Create an IPv4 network i.e. ```ipv4-network-1 = 192.168.10.0/24``` attached to ATE port-1 and port-2
+*   Create an IPv6 network i.e. ```ipv6-network-1 = 2024:db8:64:64::/64``` attached to ATE port-1 and port-2
+*   Configure IPv4 and IPv6 eBGP between DUT Port-1 and ATE Port-1
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/config
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/afi-safis/afi-safi/config/
+    *   Advertise ```ipv4-network-1 = 192.168.10.0/24``` and ```ipv6-network-1 = 2024:db8:64:64::/64``` from ATE to DUT over the IPv4 and IPv6 eBGP session on port-1
+*   Configure IPv4 and IPv6 IS-IS between DUT Port-2 and ATE Port-2
+    *   /network-instances/network-instance/protocols/protocol/isis/global/config
+    *   Advertise ```ipv4-network-1 = 192.168.10.0/24``` and ```ipv6-network-1 = 2024:db8:64:64::/64``` from ATE to DUT over the IPv4 and IPv6 IS-IS session on port-1
+
+### RT-1.34.1 [TODO:https://github.com/openconfig/featureprofiles/issues/3050]
+#### Validate traffic with default Route-Distance of eBGP
```

Comment:
It appears that some assumptions are made about the default distances based on a specific vendor implementation. 

OC doesn't define the defaults, so I'd suggest to use explicit config (e.g. 5 for eBGP in this example)

Reply:
Agree that we should explicitly configure 
BGP [default-route-distance](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-protocols-protocol-bgp-global-default-route-distance) and ISIS [config/external-route-preference](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-protocols-protocol-isis-levels-level-route-preference-config-external-route-preference)

to guarantee the behavior we're looking for since distance/preference is not a formal standard.

Reply:
Ok, so I am going to use BGP distance of 5 and 250 for the tests. I want to avoid configuring ISIS distance which is not the intent of the test. Distance of 5 and 250 will work for all vendors, ISIS falls between these values only

Reply:
That sounds good enough to me.